### PR TITLE
Add channel linking to list of keyboard shortcuts

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/help/messaging/keyboard-shortcuts.rst [hunk 1]
```diff
@@ -49,6 +49,8 @@ Messages
 +----------------------------------------+----------------------------------------+----------------------------------------------------------------------------+
 | @[character]+TAB                       | @[character]+TAB                       | Autocomplete @username beginning with [character]                          |
 +----------------------------------------+----------------------------------------+----------------------------------------------------------------------------+
+| #[character]+TAB                       | #[character]+TAB                       | Autocomplete channel beginning with [character]                            |
```
Comment (esethna) [inline source/help/messaging/keyboard-shortcuts.rst]: Isn't it a tilda `~`?

**Comment by esethna** [comment]
+1, thanks!

**Comment by jasonblais** [comment]
Not sure whether we're going to hold off on marketing channel linking -- if so, we should punt this PR to 3.6.

**Comment by jasonblais** [comment]
Sounds like we'll add this for docs, so no longer on hold

**Comment by yangchen1** [comment]
+1

**Pull request merged**